ios/RCTTWVideoModule.m

```diff
@@ -86,33 +86,37 @@ - (void)removeParticipantView:(TVIVideoView *)view identity:(NSString *)identity
 
 - (void)addParticipantView:(TVIVideoView *)view identity:(NSString *)identity  trackId:(NSString *)trackId {
   // Lookup for the participant in the room
-  for (TVIParticipant *participant in self.room.participants) {
+  for (TVIRemoteParticipant *participant in self.room.remoteParticipants) {
     if ([participant.identity isEqualToString:identity]) {
-
       // Lookup for the given trackId
-      for (TVIVideoTrack *videoTrack in participant.videoTracks) {
-        [videoTrack addRenderer:view];
+      for (TVIRemoteVideoTrackPublication *videoTrackPublication in participant.videoTracks) {
+        [videoTrackPublication.videoTrack addRenderer:view];
       }
     }
   }
 }
 
+- (void)logMessage:(NSString *)msg {
```

Comment:
If we are going to add logging, user's should be able to turn it on/off. Maybe we can do this in a separate PR and remove the logging for the moment and just focus on 2.0